### PR TITLE
Dts 25561 field mapping distinct config

### PR DESCRIPTION
**Pull request by Chittauri**

### CODE REVIEW

**Review by BhagyashreeShettigar — DISMISSED** [review]

File: database-scripts/scripts/metadata_identifier.js
Comment (Chittauri) [inline database-scripts/scripts/metadata_identifier.js:1]: revert the changes

File: database-scripts/scripts/kpi_fieldmapping_insert.js
Comment (Chittauri) [inline database-scripts/scripts/kpi_fieldmapping_insert.js:1]: 1. delete the same script from kpi_definition.js under installation script
2. add fieldmapping_structure in kpi_definition.js under installation script.
3. add fieldmapping_structure and fieldmapping_configuration under 6-7 upgrade script, remove u[other script related to kpi_fieldmapping

**Review by purushottam08 — APPROVED** [review]

**Review by brahmanand1 — APPROVED** [review]
